### PR TITLE
Docs: Add Thoth Tech marketing resources

### PR DESCRIPTION
**Pull request by ginge270**
This PR contains marketing files from T1 2022.

### CODE REVIEW

File: docs/company/marketing/pitch.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+We value our people and we value excellence: We are people-focused, aim to produce products of excellent quality, sustainability and to provide frameworks that provide a safe environment for learning and support of team-members
+
+- A chance to develop your technical skills and understanding across multiple disciplines which will make you a better version of whatever role is in your career roles
+- As well as applying crucial soft skills and understanding of how different roles within a company interact and collaborate
+- Focus on learning, support and mentoring
+
+## Products
+
+- **Ontrack**: Ontrack is an education platform to connect tutors and students at Deakin university and is also used at other universities around the world
+- **Splashkit**: Splashkit is a c++ Software Development Kit. Splashkit is an all purpose beginners kit and can be used to create 2D games
```
Comment (tancnle) [inline docs/company/marketing/pitch.md]: **nitpick:** To align with the tagline on the SplashKit [homepage](https://splashkit.io/).

```suggestion
- **SplashKit**: SplashKit is a C++ Software Development Kit. It is a beginner's all-purpose software toolkit and can be used to create 2D games.
```

File: docs/company/marketing/recruitment-ad.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+If you have any questions, please don't hesitate to contact the Thoth Tech Leadership Team by email or MS Teams chat:
+
+Managing Director (Staff Leader): Andrew Cain - andrew.cain@deakin.edu.au
+Managing Director (Staff Leader) Glory Lee - glory.lee@deakin.edu.au
+
+**QA Lead** (Student Leader) Katrina Steen - ksteen@deakin.edu.au
+**Tech Learning Lead** (Student Leader) Liz Lynch - lynchel@deakin.edu.au
+**People & Documentation Lead** (Student Leader) Talia Zidar - tzidar@deakin.edu.au
+**Data Lead** (Student Leader) Tan Le - tanle@deakin.edu.au
```
Comment (tancnle) [inline docs/company/marketing/recruitment-ad.md]: **suggestion:** Could we turn this into an unordered list? They are rendered as one line at the moment 🤔

File: docs/company/marketing/pitch.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+We value our people and we value excellence: We are people-focused, aim to produce products of excellent quality, sustainability and to provide frameworks that provide a safe environment for learning and support of team-members
+
+- A chance to develop your technical skills and understanding across multiple disciplines which will make you a better version of whatever role is in your career roles
+- As well as applying crucial soft skills and understanding of how different roles within a company interact and collaborate
+- Focus on learning, support and mentoring
+
+## Products
+
+- **Ontrack**: Ontrack is an education platform to connect tutors and students at Deakin university and is also used at other universities around the world
```
Comment (tancnle) [inline docs/company/marketing/pitch.md]:
```suggestion
- **Ontrack**: Ontrack is an education platform to connect tutors and students at Deakin university and is also used at other universities around the world.
```

File: docs/company/marketing/pitch.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+- **Splashkit**: Splashkit is a c++ Software Development Kit. Splashkit is an all purpose beginners kit and can be used to create 2D games
+
+The focus this trimester is on new feature design and development, existing feature upgrades, integrations, company planning, testing, data analysis and innovation, documentation and building a pipeline
+
+## Current Tech Stack
+
+Ruby on Rails , Rabbit MQ , Docker, Angular, C/C++, Typescript, Bootstrap, Coffeescript, Python, Cpp.
```
Comment (tancnle) [inline docs/company/marketing/pitch.md]:
```suggestion
Ruby on Rails, Rabbit MQ, Docker, Angular, C/C++, TypeScript, Bootstrap, CoffeeScript, Python, Cpp.
```

File: docs/company/marketing/pitch.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+## Recruiting
+
+large range of roles including Front-End Engineers, Back-End Engineers, Full-Stack Engineer, Data Scientists, Product Analysts, Quality Analysts, Business Analysts, Delivery Leads, Cloud Engineers, Architects, DevOps Engineers, Security Analysts, and a variety of leadership roles
```
Comment (tancnle) [inline docs/company/marketing/pitch.md]:
```suggestion
We are looking for a large range of roles including Front-End Engineers, Back-End Engineers, Full-Stack Engineers, Data Scientists, Product Analysts, Quality Analysts, Business Analysts, Delivery Leads, Cloud Engineers, Architects, DevOps Engineers, Security Analysts, and a variety of leadership roles.
```

**Review by tancnle — COMMENTED** [review]
Thanks, @ginge270. I have left some suggestions for your consideration. Please let me know what you think

File: docs/company/marketing/recruitment-ad.md (hunk 2 above)
Comment (ginge270) [reply docs/company/marketing/recruitment-ad.md]: absolutely - will do now

**Review by tancnle — APPROVED** [review]
Thanks @ginge270 🚀

**Pull request merged**